### ISSUE TITLE
Maintainer NEEDED

### ISSUE THREAD

**Issue by nadchif** [nadchif/html-duration-picker.js#156]
Looking for someone to help maintain this picker.

Must be someone passionate about open source and with a 
- Good understanding of javascript and html.
- A good idea on different browser behaviors

Recently been busy with work and school it's been hard to give the right attention for this project.

Are you interested? Please comment on this issue and let's get in touch

****

**Comment by acenasir** [comment]
I would love to take on this Issue as Maintainer. I have extensive HTML/Java Exp :)

**Comment by joyreuben** [comment]
I am in interested in this issue. I am good in JavaScript and HTML

**Comment by dei-layborer** [comment]
Is this project still being worked on?  I'd be glad to pitch in, but it doesn't look like there's been much if any activity since last year.

**Comment by abdulriiyaz** [comment]
@dei-layborer You're right, I don't think this will be needing a MAINTAINER anytime soon.

**Comment by aboodJS** [comment]
I can take the project, I am good enough in HTML and JS and will be able to maintain the project for the future
